content/en/docs/rules/overriding.md

```diff
@@ -0,0 +1,273 @@
+
+### Append an item to a list
+
+##### `/etc/falco/falco_rules.yaml`
+
+```yaml
+- list: my_programs
+  items: [ls, cat, pwd]
+
+- rule: my_programs_opened_file
+  desc: track whenever a set of programs opens a file
+  condition: proc.name in (my_programs) and (evt.type=open or evt.type=openat)
+  output: a tracked program opened a file (user=%user.name command=%proc.cmdline file=%fd.name)
+  priority: INFO
+```
+
+##### `/etc/falco/falco_rules.local.yaml`
+
+```yaml
+- list: my_programs
+  items: cp
+  override:
+    items: append
+```
+
+The rule `my_programs_opened_file` would trigger whenever any of `ls`, `cat`, `pwd`, or `cp` opened a file.
+
+### Replace items in a list
+
+##### `/etc/falco/falco_rules.yaml`
+
+```yaml
+- list: my_programs
+  items: [ls, cat, pwd]
+
+- rule: my_programs_opened_file
+  desc: track whenever a set of programs opens a file
+  condition: proc.name in (my_programs) and (evt.type=open or evt.type=openat)
+  output: a tracked program opened a file (user=%user.name command=%proc.cmdline file=%fd.name)
+  priority: INFO
+```
+
+##### `/etc/falco/falco_rules.local.yaml`
+
+```yaml
+- list: my_programs
+  items: vi, vim, nano 
+  override:
+    items: replace
+```
+The rule `my_programs_opened_file` would trigger whenever any of `vi`, `vim`, or `nano` opened a file.
+
+### Append an item to a macro
+
+##### `/etc/falco/falco_rules.yaml`
+
+```yaml
+- macro: access_file
+  condition: evt.type=open
+
+- rule: program_accesses_file
+  desc: track whenever a set of programs opens a file
+  condition: proc.name in (cat, ls) and (access_file)
+  output: a tracked program opened a file (user=%user.name command=%proc.cmdline file=%fd.name)
+  priority: INFO
+```
+
+##### `/etc/falco/falco_rules.local.yaml`
+```yaml
+- macro: access_file
+  condition: or evt.type=openat
+  override:
+    condition: append
+
+```
+
+The rule `program_accesses_file` would trigger when `ls`/`cat` either used `open`/`openat` on a file.
+
+### Append and replace items in a rule
+
+##### `/etc/falco/falco_rules.yaml`
+```yaml
+- rule: program_accesses_file
+  desc: track whenever a set of programs opens a file
+  condition: proc.name in (cat, ls) and evt.type=open
+  output: a tracked program opened a file (user=%user.name command=%proc.cmdline file=%fd.name)
+  priority: INFO
+```
+
+##### `/etc/falco/falco_rules.local.yaml`
+
+```yaml
+- rule: program_accesses_file
+  condition: and not user.name=root
+  output: A file (user=%user.name command=%proc.cmdline file=%fd.name) was opened by a monitored program
+  override: 
+    condition: append
+    output: replace
+```
+The rule `program_accesses_file` would trigger when `ls`/`cat` either used `open` on a file, but not if the user was root.
+
+The new output message would be `A file (user=%user.name command=%proc.cmdline file=%fd.name) was opened by a monitored program`
+
```

Comment:
Just an idea...maybe we can add yet another example on the `enabled` case.
I mean, something like this:

```yaml
- rule: test_rule
  desc: test rule description
  condition: evt.type = close
  output: user=%user.name command=%proc.cmdline file=%fd.name
  priority: INFO
  enabled: false

- rule: test_rule
  enabled: true
```

is now deprecated and you should use instead:


```yaml
- rule: test_rule
  desc: test rule description
  condition: evt.type = close
  output: user=%user.name command=%proc.cmdline file=%fd.name
  priority: INFO
  enabled: false

- rule: test_rule
  enabled: true
  override:
    enabled: replace
```

In this way we show the new correct usage of `enabled` WDYT?

Reply:
and also in this example, we can highlight that this usage is deprecated! Falco will output a warning if you use it
```yaml
- rule: test_rule
  desc: test rule description
  condition: evt.type = close
  output: user=%user.name command=%proc.cmdline file=%fd.name
  priority: INFO
  enabled: false

- rule: test_rule
  enabled: true
```

Reply:
I made all the changes you suggested @Andreagit97  Can your please review. Thanks!